src/main/java/com/foundation/salesforce/core/api/AccountApi.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * @(#) AccountApi.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package com.foundation.salesforce.core.api;
+
+import com.foundation.salesforce.core.restClient.Authentication;
+import com.foundation.salesforce.core.restClient.RestClientApi;
+import io.restassured.response.Response;
+
+import static com.foundation.salesforce.core.utils.EndPoints.ACCOUNT_ENDPOINT;
```

Comment:
Please, place this line in the same group, i.e. after import com.foudnation.salesforce.core.restClient. ... group